utils/questions.py

```diff
@@ -1,10 +1,286 @@
 from typing import Any
 
 import requests
+import random
 
 from bot_globals import logger
 from embeds.misc_embeds import error_embed
+from embeds.questions_embeds import question_embed
+from utils.run_blocking import to_thread
+from utils.ratings import get_rating_data
+import re
```

Comment:
in vs code, press ctrl+shift+p, then type organize imports. Do this for all files you've modified.